core/src/test/java/tachyon/hadoop/fs/TestDFSIO.java

```diff
@@ -795,20 +814,23 @@ else if (testType == TestType.TEST_TYPE_READ_SKIP && skipSize == 0)
 
   @Override
   // Configurable
-  public Configuration getConf() {
+      public
+      Configuration getConf() {
```

Comment:
Please format this file. https://github.com/amplab/tachyon/blob/master/docs/resources/tachyon-code-formatter-eclipse.xml